### PR TITLE
Auth routing setup

### PR DESCRIPTION
**Pull request by ArshdeepSingh3-eaton**
Fixes # .

#### Changes proposed in this Pull Request:

- Updated the route to implement Registration Workflow
-
-

#### Screenshots / Screen Recording (if applicable)

-

#### To Test:

- in example/src/navigation/index.tsx
change 
```
const getAuthState = (): any => ({
    **isAuthenticated: false,**
});
```

#### Any specific feedback you are looking for?

-

### CODE REVIEW

**Review by surajeaton — COMMENTED** [review]
Please resolve the conflicts.

File: login-workflow/example/src/navigation/index.tsx [hunk 1]
```diff
@@ -36,31 +40,63 @@ const CustomDrawerContent = (props: any): any => (
...
+    >
+        <RootStack.Screen name="Home" component={Home} />
+        <RootStack.Screen
+            name="RegistrationProviderExample"
+            component={RegistrationProviderExample}
+            options={{ gestureEnabled: false }}
+        />
+    </Drawer.Navigator>
 );
+const AuthRouter = (): any => {
```
Comment (ektaghag-eaton) [inline login-workflow/example/src/navigation/index.tsx:61]: this should be registrationRouter

**Review by ektaghag-eaton — COMMENTED** [review]
https://github.com/etn-ccis/blui-react-native-workflows/pull/396/files#diff-08acea08d1e5a45bf800e752666fef1e3f1bd3a77133a5a70f23e1da73aea5f5R25

Remove unused path

**Comment by ektaghag-eaton** [comment]
Please check the above comments. Other than this changes look good 👍 .

**Review by ektaghag-eaton — APPROVED** [review]

**Pull request merged**